clang/docs/ReleaseNotes.rst

```diff
@@ -960,6 +960,7 @@ Sanitizers
 ----------
 
 - ``-fsanitize=vptr`` is no longer a part of ``-fsanitize=undefined``.
+- UBSan ignorelists now support the syntax ``src:*=sanitize``.
```

Comment:
I think it applies to all Sanitizers, with src: syntax

Reply:
Yes, it should work for all sanitizer. However, I don't have tests for all other sanitizers. I don't feel confident to claim that the syntax should apply to all sanitizers.